### ISSUE TITLE
Deploy not working on Windows

### ISSUE THREAD

**Issue by rollinball** [graphcool/framework#885]
**Current behavior**
When requested the 'hello(name)' function returns null

**Reproduction**

Request:
`query {
  hello(name:"TEST") {
    message
  }
}`
Result:
`{
  "data": {
    "hello": null
  }
}
`

**Expected behavior?**

Should result in:
`{
  "data": {
    "hello": {message: "Hello TEST"}
  }
}`

**Comment by marktani** [comment]
Hey @rollinball, I just did the following with `graphcool@0.7.0` and could not reproduce the problem:

```
graphcool init
graphcool deploy
```

This `hello` query works as expected:

```graphql
query {
  hello {
    message
  }
}
```

```json
{
  "data": {
    "hello": {
      "message": "Hello World"
    }
  }
}
```

Can you please include more information that helps reproduce this? Can you paste the output of `graphcool version`, too?

**Comment by rollinball** [comment]
hi Mark
thanks for the reply.
well if i do just what you did:
graphcool init test
cd test
graphcool deploy

Then on playground i do 
query {
  hello {
    message
  }
}
I hit play
and I get
{
  "data": {
    "hello": null
  }
}

graphcool version
graphcool/0.7.0 (windows-x64) node-v6.11.4

I'm on Win 10.

**Comment by marktani** [comment]
Hey there, I'm pretty sure you were running into this: https://github.com/graphcool/graphcool/issues/896

Can you confirm?

By the way, I'm Nilan, not Mark 😄

**Comment by rollinball** [comment]
Hi Nilan,
Sorry about that, I myself do return null sometimes, those new technologies are very disturbing.

it seems to be related, tho I didn't change anything.
here is my log:
`{"event":"{\"data\":{\"name\":\"test\"},\"context\":{\"request\":{\"sourceIp\":\"77.139.195.113\",\"headers\":{},\"httpMethod\":\"post\"},\"auth\":null,\"sessionCache\":{},\"environment\":{},\"graphcool\":{\"rootToken\":\"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1MDgzOTUzNTUsImlhdCI6MTUwODM5NTA1NSwiY2xpZW50SWQiOiJfIiwicHJvamVjdElkIjoiY2o4d3NiMno1MGg1eDAxNDFlbjFudmY5byIsInBlcm1hbmVudEF1dGhUb2tlbklkIjoiY2o4eTNneHRtNzFwdjAxMjIwdDZ6YzJoYSJ9._2N-WMOPNi048OT3v7LNFZUDmKi-ynUe7YLqHQcZi48\",\"endpoints\":{\"simple\":\"https://api.graph.cool/simple/v1/cj8wsb2z50h5x0141en1nvf9o\",\"relay\":\"https://api.graph.cool/relay/v1/cj8wsb2z50h5x0141en1nvf9o\",\"system\":\"https://api.graph.cool/system\",\"subscriptions\":\"wss://subscriptions.graph.cool/v1/cj8wsb2z50h5x0141en1nvf9o\"},\"projectId\":\"cj8wsb2z50h5x0141en1nvf9o\",\"alias\":null,\"pat\":\"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJleHAiOjE1MDgzOTUzNTUsImlhdCI6MTUwODM5NTA1NSwiY2xpZW50SWQiOiJfIiwicHJvamVjdElkIjoiY2o4d3NiMno1MGg1eDAxNDFlbjFudmY5byIsInBlcm1hbmVudEF1dGhUb2tlbklkIjoiY2o4eTNneHRtNzFwdjAxMjIwdDZ6YzJoYSJ9._2N-WMOPNi048OT3v7LNFZUDmKi-ynUe7YLqHQcZi48\",\"serviceId\":\"cj8wsb2z50h5x0141en1nvf9o\"}}}","logs":["START RequestId: fe4538c0-b497-11e7-8d81-95426181e9ac Version: $LATEST\nUnable to import module 'src\\hello-lambda': Error\n    at Function.Module._resolveFilename (module.js:469:15)\n    at Function.Module._load (module.js:417:25)\n    at Module.require (module.js:497:17)\n    at require (internal/module.js:20:19)\nEND RequestId: fe4538c0-b497-11e7-8d81-95426181e9ac\nREPORT RequestId: fe4538c0-b497-11e7-8d81-95426181e9ac\tDuration: 36.41 ms\tBilled Duration: 100 ms \tMemory Size: 512 MB\tMax Memory Used: 20 MB\t\n"],"returnValue":{"errorMessage":"Cannot find module '/var/task/src\\hello-lambda'","errorType":"Error","stackTrace":["Function.Module._load (module.js:417:25)","Module.require (module.js:497:17)","require (internal/module.js:20:19)"]}}`

**Comment by lolcoolkat** [comment]
Bumping this up! I'm getting the same issue :)

**Comment by kbrandwijk** [comment]
I can confirm that the deploy is still not working on Windows with the error from https://github.com/graphcool/graphcool/issues/885#issuecomment-337814173.

**Comment by kbrandwijk** [comment]
Can we update the title to 'Deploy not working on Windows' and add a 'breaking' label?

**Comment by marktani** [comment]
> 'Deploy not working on Windows' 

We identified a couple of issues  (see https://github.com/graphcool/graphcool/labels/theme%2Fcli-windows and more that are not documented yet) and will add windows support soon. It will probably be released sometime leading up to version 0.10.

> and add a 'breaking' label?

Side question, why is this breaking?

**Comment by kbrandwijk** [comment]
> Side question, why is this breaking?
Because deploy on Windows is no longer possible in the _released_ Framework Preview version? That was fine when it was still alpha, but the new function deployment is released as `@next`.

**Comment by marktani** [comment]
The `breaking` label signifies that adding the feature or fixing the bug is a breaking change.
The 'breaking' label does not signify a 'regression' or the introduction of new bugs.

This issue is not a breaking change, it's a feature request that can be added additively. We'll add explicit support for Windows soon, as mentioned above. 🙂

**Comment by kbrandwijk** [comment]
Okay, so I was wrong about the `breaking` label, but the fact that Windows deployment stopped working from `@next 0.7.0` to `@next 0.8.0` is still a valid argument I think.

Especially because downgrading to 0.7.0. doesn't help, as the change is on the server-side.

**Comment by lolcoolkat** [comment]
Hope this can be fixed really soon! It's put me at a stand still for doing anything function related via the cli :)

**Comment by lolcoolkat** [comment]
@marktani any way this can be bumped up to 0.9?

**Comment by kbrandwijk** [comment]
Update, as Windows deployment is still not working. Developers that use Windows 10 can take advantage of a feature called 'Windows Subsystem for Linux'. It basically runs a native Linux bash shell, with your Windows filesystem mounted. So you can easily use that shell to deploy.

**Comment by marktani** [comment]
This is now released in `0.9.1`: https://github.com/graphcool/framework/releases/tag/0.9.1 🙂
